src/test/core/Config_test.cpp

```diff
@@ -201,9 +210,11 @@ class RippledCfgGuard : ConfigGuard
 
 public:
     RippledCfgGuard (beast::unit_test::suite& test,
-        std::string subDir, std::string const& dbPath,
-            std::string const& validatorsFile)
-        : ConfigGuard (test, std::move (subDir)), dataDir_ (dbPath)
+        path subDir, path const& dbPath,
```

Comment:
I do not in this case, because `configContents` is using those params to build a string, and it doesn't make sense to me to force the input to be a `path` when we only want a string. If it was more widely used, and not just test code, I'd argue that a caller might want to just pass in a string, and promoting it to `path` only to just use a string is unnecessary.

I am going to take the rest of your `const` changes, though. Those are nice!